Automattic/vip
# Adds check for AUTO_INCREMENT attribute in SQL import

## Description

This PR adds updates the list of validation checks during SQL import to include the check to validate the presence of `AUTO_INCREMENT` attribute in the `CREATE TABLE` statements in the SQL file.

## Pull request checklist


## New release checklist


## Steps to Test

1. Check out the PR
2. Run `npm run test:e2e:dev-env` to test the validation check for `AUTO_INCREMENT` attribute
